src/lib/components/search/display/images/Images.svelte

```diff
@@ -14,14 +14,21 @@
 	export let paramsString: string;
 
 	// variables
-	let imgResultPreview: ResultType | undefined;
+	export let imgResultPreview: ResultType | undefined;
+
+	// helper function
+	function closeIt(e: any) {
+		if (e.keyCode == 27) {
```

Comment:
Use === instead of ==, also is any type necessary? If you can't determine what type is needed it's ok to be any, but if you can it's better to use that.